### DISCUSSION TITLE
Move @swc/core to dev dependencies of react-email

### DISCUSSION THREAD

**Discussion by skelouse** [resend/react-email#1388] [General]
### Describe the Bug

Added in [this commit](https://github.com/resend/react-email/commit/ce7454c5a0fb5d30a5642cd7b36bed41d4f8a13a) I'm not sure why this is added to dependencies and not dev dependencies.

I also found that running `pnpm run build` here with the package removed requests you to install it with `-D`

**Comment by gabrielmfern** [comment]
As mentioned on another few discussions, we do build the preview app before publishing it. At least not yet. It just isn't that trivial as it may seem with the way we are doing things.
So once you run `email dev` Next.js will actually compile the preview app. This is why, if we include it as a dev dependency, it might not be included into the user's installation which will break the `email dev` command.

Will be converting this into a discussion.

**Reply by skelouse** [reply]
Thanks for the quick response, I noticed this when reaching vercel edge function size limits.  I ignore the problem files with the following config:

`next.config.js`
```js
  experimental: {
    outputFileTracingIgnores: [
      '**/node_modules/@swc/core-linux-x64-gnu',
      '**/node_modules/@swc/core-linux-x64-musl',
    ],
  },
  ```
  
I made this issue originally to solve this in a better way, as except for `email dev` the user does not need this large package.

For the nextjs use case `@swc/core` is listed as a package that is working on an auto opt-out: https://nextjs.org/docs/app/api-reference/next-config-js/serverComponentsExternalPackages
